docs/questions/native-editor/sql-parameters.md

```diff
@@ -324,16 +221,125 @@ WHERE
 {% endraw %}
 ```
 
-### SQL syntax
+## Customizing dropdown lists and search box values
+
+With Text and Field filter variables, you can tell Metabase what values people can choose from when using a filter with a dropdown list or search box.
 
-Make sure your SQL dialect matches the database you've selected. Common errors:
+1. In the native editor, add a {% raw %}{{variable}}{% endraw %} in double braces.
+2. If the sidebar doesn't open, you can click on the **{x}** icon on the right to open the **Variables** sidebar.
+3. In the **Settings** tab, set the **Variable type** to either "Text" or "Field Filter".
+4. Scroll down to **How should users filter on this variable?** Pick either **Dropdown list** or **Search box**.
+5. Next to the option you chose, click **Edit**.
+6. Metabase will pop up a modal where you can select **Where the values should come from**.
 
-| Database | Do this                    | Avoid                |
-| -------- | -------------------------- | -------------------- |
-| BigQuery | `` FROM `dataset.table` `` | `FROM dataset.table` |
-| Oracle   | `FROM "schema"."table"`    | `FROM schema.table`  |
+You can choose:
 
-For more help, see [Troubleshooting SQL error messages](../../troubleshooting-guide/error-message.md#sql-editor).
+- A custom list. Enter each item on a line.
```

Comment:
I think this should follow the second one "A model or saved question", as that is the sequence presented in the app 
<img width="972" alt="Screenshot 2024-03-11 at 12 18 14 PM" src="https://github.com/metabase/metabase/assets/154609371/b13b52b9-315f-4b12-9138-9e87b508ed92">